client/my-sites/plan-features-list/style.scss

```diff
@@ -0,0 +1,11 @@
+.plan-features-list {
+	@include breakpoint( ">1040px" ) {
+		display: flex;
+		flex-flow: row wrap;
+		padding-top: 19px; // popular banner height adjustment
+	}
+
+	+ .faq {
```

Comment:
We try not to nest, though the sibling selector here might be OK. https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#nesting